### DISCUSSION TITLE
Asking for update and then completely malfunctioning

### DISCUSSION THREAD

**Discussion by 685685** [valinet/ExplorerPatcher#1811] [General]
When I started my computer this morning, I got the message about an update about 10 times in the lower right corner of my screen. My task bar keeps flashing and isn't working anymore. I tried to update, but then the flashing is still there and the ExplorerPatcher shuts down at some point.

**Comment by LittlDog** [comment]
Had the same Problem. Seems the last Windows Updates broke something again. 
2023-08 Kumulatives Update für .NET Framework 3.5 und 4.8.1 für Windows 11, version 22H2 für x64 (KB5028948)
2023-08 Kumulatives Update für Windows 11 Version 22H2 für x64-basierte Systeme (KB5029263)
But after Logging out of my Windows Acc and logging back in, FE is working now again.
Don't know why.

**Comment by pyrates999** [comment]
You may have not waited until symbols were downloaded.  To make sure explorer patcher uses them, after you get the notification that symbols were downloaded, reboot.
